skipper/runner.py

```diff
@@ -37,7 +38,12 @@ def _run_nested(fqdn_image, environment, command, interactive):
         docker_cmd += ['-e', env]
 
     user = os.environ.get('USER', 'skipper')
+    if user != 'skipper':
+        user_uid = pwd.getpwnam(user).pw_uid
+    else:
+        user_uid = '1000'
```

Comment:
more simple method:
```python
user_id = os.getuid()
```
note that i used `user_id` and not `user_uid` (the 2nd u is redundant).